src/app/item-popup/ItemDescription.m.scss

```diff
@@ -11,9 +11,14 @@
 
 .addNote {
   cursor: pointer;
+  margin-top: 0 !important;
+  margin-bottom: 0 !important;
   &:hover {
     color: $orange;
   }
+  &.noNotesYet {
+    display: inline-block;
+  }
```

Comment:
With CSS modules you don't really need to nest selectors

Reply:
unless it's a performance impact, i prefer to because it helps outline intended hierachy in the scss file, and affects selector precedence

with class nesting the inner will take precedence if an attribute needs overriding, because it's a 2 class selector.

briefly this was
```
.class1{
  display: inline-block;
  .override{
    display: block
  }
}
```

and
```
.class1{
  display: inline-block;
}
.override{
  display: block
}
```
would have worked but seems a bit more fragile due to order mattering, and less explicit

then it because moot since i swapped which one was the override but i still feel safer with explicit rules.